src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -379,7 +379,7 @@ class DataBox<tmpl::list<Tags...>>
 
   /// A list of all the compute item tags, excluding their subitems
```

Comment:
Looking through this commit, I see that in the comments and documentation for the Databox the phrases "compute item", "simple item", and "subitem" appear many times. Ultimately, do we want to keep calling them by these phrases? Or are these more properly called "compute tags", "simple tags", and ___?

I ask this simply out of ignorance. But if we do want to make the broader naming change, and you didn't want to make it in this PR, an issue might be good to remind us to revisit this.

Reply:
So I consider a "_ tag" to be the stateless struct which is what this metafunction is referring to, while I consider a "_ item" to be the data to which the struct refers to in the DataBox (i.e. analogous to key and value of a map).  I am sure this is not consistently done in the documentation.   I am slowly making a number of changes to DataBox, and my plan was to make a pass through the documentation when those are finished to clean it up.

Reply:
This distinction makes sense, as does your plan for revisiting the documentation after making code changes. Thanks for clarifying! 👍